kf_lib_data_ingest/etl/load/load.py

```diff
@@ -1,5 +1,7 @@
 """
```

Comment:
Maybe for another PR but, do you think we should have an organization like this for load stage now? 

`kf_lib_data_ingest/etl/load/kids_first/load.py`

Reply:
no, why?

Reply:
Isn't this loader Kids First specific? And we will have a FHIR loader too right? So either rename the module or put it in its own package along with the message packer

Reply:
> Isn't this loader Kids First specific?

Not if the target API config gets refactored like I want

Reply:
It remains to be seen whether any of the actual loader code will need to be generalized

Reply:
Ok gotcha 👍